src/puppeteer_crawler.js

```diff
@@ -5,10 +5,9 @@ import BasicCrawler from './basic_crawler';
 import PuppeteerPool from './puppeteer_pool';
 import { addTimeoutToPromise } from './utils';
 import { BASIC_CRAWLER_TIMEOUT_MULTIPLIER } from './constants';
-import { gotoExtended } from './puppeteer_utils';
 
 const DEFAULT_OPTIONS = {
-    gotoFunction: async ({ request, page }) => gotoExtended(page, request, { timeout: 60000 }),
+    gotoFunction: async ({ request, page }) => page.goto(request.url, { timeout: 60000 }),
```

Comment:
This is incompatible change because properties `headers` and `method` of `Apify.Request` object will stop working...

Reply:
I'd keep `gotoExtended` as it does not intercept requests as long as you dont use headers and method. And if you do then we would break it to the user...

Reply:
Fair point. I'll add the warning message after the method and header check so that it gets printed only to users who override either headers or method and will keep the function there.